src/Tgstation.Server.Host/Components/Byond/ByondInstallerBase.cs

```diff
@@ -18,6 +18,11 @@ abstract class ByondInstallerBase : IByondInstaller
 		/// </summary>
 		const string CacheDirectoryName = "cache";
 
+		/// <summary>
+		/// The first <see cref="Version"/> of BYOND that supports the '-map-threads' parameter on DreamDaemon.
+		/// </summary>
+		public static Version MapThreadsVersion => new (515, 1605);
```

Comment:
The magic number. I noticed Lummox put out 1605 bins already and they don't have -map-threads so this is subject to change.

Reply:
and again

Reply:
Lummox said 1609